src/remote-config/condition-evaluator-internal.ts

```diff
@@ -145,18 +146,26 @@ export class ConditionEvaluator {
 
     const seedPrefix = seed && seed.length > 0 ? `${seed}.` : '';
     const stringToHash = `${seedPrefix}${context.randomizationId}`;
-    const hash64 = Math.abs(parseFloat(farmhash.fingerprint64(stringToHash)));
 
-    const instanceMicroPercentile = hash64 % (100 * 1_000_000);
 
+    let hash64 = long.fromString(farmhash.fingerprint64(stringToHash));
```

Comment:
Minor: in addition to the PR description, I think it'd be worth a comment here explaining we're using a 64 bit long for consistency with the fetch endpoint.